frontend/src/images/bootstrap-search.js

```diff
@@ -11,11 +11,11 @@ const bootstrapSearch = () => {
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
-        fill-rule="evenodd"
+        fillRule="evenodd"
```

Comment:
What is this doing?

Reply:
Not sure, I think it has to do with SVG formats; just changed it because it was throwing a warning (because React prefers camel-cased params)